README.md

```diff
@@ -8,6 +8,18 @@ The vlingo/PLATFORM  central "build all" and other control.
 
 [Read about the vlingo/platform](https://kalele.io/blog-posts/vlingo-platform/) and about [its architecture](https://kalele.io/blog-posts/vlingo-platform-architecture-part1/).
 
+
+## Getting started
+
+Prerequisites:
+* Java JDK 8
+
+To check out all `vlingo-platform` modules, ...
+* ... create a parent directory to hold all your `vlingo` repos
+* ... `cd` into the new repo
```

Comment:
I think this should be:

To check out and build all `vlingo-platform` modules, do the following:
```
$ cd 'my-projects-root-directory'
$ mkdir vlingo
$ cd vlingo
$ git clone https://github.com/vlingo/vlingo-platform.git
$ cd vlingo-platform
$ ./gradlew
$ mvn install
```